net/netclient/patches/010-install-initd-fix.patch

```diff
@@ -0,0 +1,59 @@
+--- a/daemon/initd_linux.go
++++ b/daemon/initd_linux.go
+@@ -1,37 +1,12 @@
+ package daemon
+ 
+ import (
+-	"errors"
+-	"os"
+-
+ 	"github.com/gravitl/netclient/ncutils"
+ 	"golang.org/x/exp/slog"
+ )
+ 
+ // setupInitd - sets up initd daemon
+ func setupInitd() error {
+-	service := `#!/bin/sh /etc/rc.common
+-	START=99
+-	STOP=99
+-	USE_PROCD=1
+-	
+-	start_service() {
+-        procd_open_instance
+-        procd_set_param command /sbin/netclient daemon
+-        procd_set_param stdout 1
+-        procd_set_param stderr 1
+-        procd_close_instance
+-}
+-`
+-	bytes := []byte(service)
+-	if err := os.WriteFile("/etc/init.d/netclient", bytes, 0755); err != nil {
+-		slog.Debug("error writing initd service file", "error", err)
+-		return err
+-	}
+-	if _, err := ncutils.RunCmd("/etc/init.d/netclient enable", false); err != nil {
+-		slog.Debug("error enabling initd service", "error", err)
+-		return err
+-	}
+ 	return nil
+ }
+ 
+@@ -58,18 +33,5 @@ func restartInitd() error {
+ 
+ // removeInitd - removes initd daemon
+ func removeInitd() error {
+-	var faults string
+-	if _, err := ncutils.RunCmd("/etc/init.d/netclient disable", true); err != nil {
+-		faults = faults + err.Error()
+-	}
+-	if ncutils.FileExists("/etc/init.d/netclient") {
+-		if err := os.Remove("/etc/init.d/netclient"); err != nil {
+-			slog.Info("Error removing /etc/init.d/netclient. Please investigate.")
+-			faults = faults + err.Error()
+-		}
+-	}
+-	if faults != "" {
+-		return errors.New(faults)
+-	}
+ 	return nil
```

Comment:
Why?

Reply:
commit message has an explanation.

Reply:
I was talking about this specific thing, why you need to remove it. "OpenWrt provides its", and in this case, you provide it. Is not sufficient at all!

Reply:
commit message has an explanation.

Reply:
Patch does not. Those are two different things.